index.html

```diff
@@ -263,7 +264,13 @@ <h3>
             <li>Presentation Request Data Model</li>
             <li>Storage and Sharing of Verifiable Credentials</li>
             <li>Privacy Guidance for Verifiable Credentials</li>
-            <li>An API for Verifiable Credential Exchange</li>
+            <li>A Developer Guide consisting of one or more notes related to:
+              <ul>
+                <li>General implementaion guidance and best practices for working with VCs</li>
+                <li>One or more HTTP protocol definitions for Verifiable Credential Exchange (such as the <a href="https://w3c-ccg.github.io/vc-api/">VC-API</a>)</li>
+                <li>Guidance on Verifiable Credential Exchange over OpenID Connect</li>
```

Comment:
```suggestion
                <li>Potential guidance on Verifiable Credential Exchange over OpenID Connect</li>
```

Reply:
Protocol definitions or *profiles* of protocols (and/or specific APIs)?

Reply:
@bumblefudge I think the thing is we want to be able to let working group members decide how far down that rabbit hole they way to go.  Some of these things are in early stages, but by the time the WG is in full swing will be much further along and there is a good chance we will need to be able to comment on these things from a dev guidance and implementation standpoint.  I would rather explicitly state that the WG may choose to address questions in these areas (especially if, as is the case, there are participants willing to dedicate resources to these types of items), than not be clear up font that we see this as an area we may need to touch on, and running into difficulties later for not being explicit on the charter regarding what might or might not be in scope.

Reply:
Since there are links to all the other protocols, links to the OIDC work also should be added (there has been an issue comment on this).
- https://openid.net/specs/openid-connect-self-issued-v2-1_0.html
- https://openid.net/specs/openid-connect-4-verifiable-presentations-1_0.html
- https://openid.net/specs/openid-connect-4-verifiable-credential-issuance-1_0.html